empire/pkg/service/ecs.go

```diff
@@ -287,6 +288,10 @@ func (m *ecsProcessManager) CreateProcess(ctx context.Context, app *App, p *Proc
 	return err
 }
 
+func (m *ecsProcessManager) Run(ctx context.Context, app *App, process *Process, in io.Reader, out io.Writer) error {
```

Comment:
Might be worth a comment here describing why this is for now - might help a couple of people (like it did me, when we discussed it :))